std_msgs/Header header
uint32 next_waypoint
racer/WaypointMsg[] waypoints